### ISSUE TITLE
I want to change selected tag's text color and border color

### ISSUE THREAD

**Issue by kkarynossang** [whilu/AndroidTagView#52]

**Comment by whilu** [comment]
Closed?

2017年5月16日 09:46，"kkarynossang" <notifications@github.com>写道：

Closed #52 <https://github.com/whilu/AndroidTagView/issues/52>.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/whilu/AndroidTagView/issues/52#event-1083319876>, or mute
the thread

**Comment by whilu** [comment]
@kkarynossang 
There is no way to get tag object with specific location,  I guess the method `getTag(int position)` which you used is the method under `View.java`:

```java
public Object getTag(int key) {
    if (mKeyedTags != null) return mKeyedTags.get(key);
        return null;
    }
}
```

this is a bit ambiguous.

If you want to change color for particular TagView, try use color list when init TagContainerLayout:

```java
List<String> list = new ArrayList<String>();
ArrayList<String> colors = new ArrayList<int[]>();

// add color for each TagView
//int[] color = {TagBackgroundColor, TabBorderColor, TagTextColor}
int[] color1 = {Color.RED, Color.BLACK, Color.WHITE};
colors.add(color1);

mTagContainerLayout.setTags(list, colors);

// when you want to change color in specific position
int[] newColor = {Color.GREEN, Color.BLACK, Color.WHITE};
colors.set(position, newColor);
mTagContainerLayout.removeTag(position);
mTagContainerLayout.addTag(list.get(position), position);
```

Hope this will help you! 

Thanks!
